build.sh

```diff
@@ -18,4 +18,10 @@ sed -e "/#INSERT_PYTHON_CODE_HERE/r ./src/aws_cli_mfa.py" -e "s///" ./src/$AWS_C
     sudo chmod +x /usr/local/bin/aws-cli-mfa &&
     cp /usr/local/bin/aws-cli-mfa ./bin/aws-cli-mfa-$AWS_CLI_MFA_SHELL
 
+sed 's/"/\\\"/g' ./src/aws_cli_mfa.py | sed -e "/#INSERT_PYTHON_CODE_HERE/r /dev/stdin" -e "s///" ./src/mfa.fish |
```

Comment:
What is this extra sed here? Is this what you were referring to above about quotes?

Reply:
Yes, this is to escape the `"` to not conflict with setting the variable.